root/static/scripts/edit/externalLinks.js

```diff
@@ -579,7 +579,7 @@ MB.createExternalLinksEditor = function (options: InitialOptionsT) {
     const urls = {};
     let match;
 
-    while (match = seededLinkRegex.exec(window.location.search)) {
+    while ((match = seededLinkRegex.exec(window.location.search))) {
```

Comment:
I don’t get how it works around `no-cond-assign`.

Reply:
https://github.com/eslint/eslint/blob/master/docs/rules/no-cond-assign.md - "except-parens" (default) allows assignments in test conditions only if they are enclosed in parentheses (for example, to allow reassigning a variable in the test of a while or do...while loop)